cairo/src/utils/utils.cairo

```diff
@@ -1075,4 +1075,263 @@ namespace Helpers {
         }
         jmp read;
     }
+
+    func pow2(i: felt) -> (res: felt) {
```

Comment:
should this be moved to python/cairo-core

Reply:
now that it exists, i guess so!